app/main.ipynb

```diff
@@ -237,18 +316,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 23,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "np.sum(A, axis=1)",
```

Comment:
The sum of elements by row is correctly calculated. Ensure that the axis parameter is set correctly for the intended operation.